subworkflows/MODEL_TO_DATA.nf

```diff
@@ -39,8 +39,8 @@ workflow MODEL_TO_DATA {
         .splitCsv(header:true) 
         .map { row -> tuple(row.submission_id, row.image_id) }
     UPDATE_SUBMISSION_STATUS_BEFORE_RUN(image_ch.map { tuple(it[0], "EVALUATION_IN_PROGRESS") })
-    BUILD_SUBFOLDERS(image_ch.map { tuple(it[0], "build") }, params.project_name, UPDATE_SUBMISSION_STATUS_BEFORE_RUN.output)
-    RUN_DOCKER(image_ch, SYNAPSE_STAGE.output, params.cpus, params.memory, BUILD_SUBFOLDERS.output)
+    CREATE_FOLDERS(image_ch.map { tuple(it[0], "build") }, params.project_name, UPDATE_SUBMISSION_STATUS_BEFORE_RUN.output)
```

Comment:
Make sure you update documentation to reflect the new process name (mermaid diagram in `README.md`)

Reply:
Good catch, thanks!